pyproject.toml

```diff
@@ -63,6 +63,9 @@ pyasn1 = "*"
 # legacy
 setuptools = ">=24.2.0"
 
+# python
```

Comment:
I did not find an entry corresponding to the Python folder, so I added it.

Reply:
You should use optional dependencies: https://python-poetry.org/docs/pyproject/#extras

`stellar-sdk = { version = ">=4.0.0,<5.0.0", optional = true }`

and then add the following at the end of the file:

```
[tool.poetry.extras]
stellar = ["stellar-sdk"]
```

Reply:
80f62f09ea2e278395de508d8e53fb6435644386

Reply:
no, given that `test_stellar.py` uses this, it should be part of the main pyproject.toml, not an extra. please revert this part

Reply:
a78c9c112688c8522704e019ec993ae294b59d2b

Reply:
@matejcik The current tests under the `python` folder use the environment provided by tox, so I think we can mark stellar-sdk as optional here.

https://github.com/trezor/trezor-firmware/pull/1745#discussion_r683903844